docs/source/management/role.rst

```diff
@@ -0,0 +1,68 @@
+Managing Roles
+###############
+A role defines a set of access control lists (ACLs). These ACLs define permissions and resource definitions.
+
+Permissions
+********************
+A permission is a three part object: application, resource type, operation
+
+Application specifies the specific domain for the resource control, for example::
+
+- rbac
+- catalog
+- approval
+- cost-management
+
+Resource type defines the resource to be controlled, for example::
+
+- group
+- role
+- policy
+- aws.account
+- openshift.cluster
+
+Operation defines the application logic action, for example::
+
+- read
+- write
+- execute
```

Comment:
@chambridge we dropped execute in favor of order, if you are not using execute we can skip it  here

Reply:
removed